subworkflows/local/input_check.nf

```diff
@@ -9,23 +9,52 @@ workflow INPUT_CHECK {
     samplesheet // file: /path/to/samplesheet.csv
 
     main:
-    SAMPLESHEET_CHECK ( samplesheet )
-        .csv
-        .splitCsv ( header:true, sep:',' )
-        .map { create_fastq_channel(it) }
-        .set { reads }
+
+    switch(params.source) {
```

Comment:
I think it'll be nice to avoid using params in the subworkflow and have that as an input

Reply:
Fixed in #68